CHANGELOG.md

```diff
@@ -11,13 +11,15 @@ All notable changes to this project will be documented in this file.
   - `affinity`
   - `volumes`
   - `volumeMounts`
+- Update tests and docs to Spark version 3.5.2 ([#459])
```

Comment:
Ideally we would mention that we dropped support for 3.4.2 and 2.4.3 and added support for 3.5.2.

Reply:
Done.